stdlib/copy.pyi

```diff
@@ -11,6 +16,10 @@ PyStringMap: Any
 def deepcopy(x: _T, memo: dict[int, Any] | None = None, _nil: Any = []) -> _T: ...
 def copy(x: _T) -> _T: ...
 
+if sys.version_info >= (3, 13):
+    def replace(obj: _SR, /, **kwargs: Any) -> _SR: ...
```

Comment:
Higher-kinded types could be cool here, but obviously no way to get great kwargs type-checking. Even in that case you can't use `P.kwargs` without `P.args`.